src/layman/layer/rest_workspace_test.py

```diff
@@ -550,12 +534,16 @@ def test_patch_layer_style():
     new_title = 'countries in blue'
 
     sld_path = 'sample/style/generic-blue_sld.xml'
-    process_client.patch_workspace_layer(workspace, layername,
-                                         title=new_title,
-                                         style_file=sld_path,
-                                         )
+    with app.app_context():
+        layer_uuid = layman_util.get_publication_uuid(workspace, 'layman.layer', layername)
+    process_client.patch_layer(uuid=layer_uuid,
+                               title=new_title,
+                               style_file=sld_path,
+                               )
 
-    get_response = process_client.get_workspace_layer(workspace=workspace, name=layername)
+    with app.app_context():
+        layer_uuid = layman_util.get_publication_uuid(workspace, 'layman.layer', layername)
```

Comment:
You can store layer uuid in global constant and publish it with it (or at least use layer_uuid from line 538)